Makefile

```diff
@@ -48,19 +48,37 @@ generate-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole
 	hack/helmify $(TEMPLATES_MANIFESTS_PATH)
 	docker run --rm -v $(shell pwd):/github/workspace $(IMG_LICENSE_EYE) -c .github/licenserc.yaml header fix
 
+.PHONY: generate-open-api-spec
+generate-generate-open-api-spec: VERSION = $(shell git rev-parse --short HEAD)
+generate-generate-open-api-spec:
```

Comment:
The second `generate` needs to be  dropped from `generate-generate-open-api-spec`.

`generate-types` calls `generate-open-api-spec` which results in `make: Nothing to be done for `generate-open-api-spec'.`, since the command has the additional `generate`

Reply:
Good catch!! I just remove the step. It seems to be a dependency for create the types to have the `openapi.yaml` from the other task.

Reply:
Did you forget to upload the removal?